### PR TITLE
Cu 862j4d59c walidacja klasy company

### PR DESCRIPTION
**Pull request by s21670-Pyrzak-Tomasz-gr-18C**

### CODE REVIEW

**Comment by s20146** [comment]
Task linked: [CU-862j4d59c walidacja klasy Company](https://app.clickup.com/t/862j4d59c)

File: Java/E-montazysta/src/main/java/com/emontazysta/model/Company.java [hunk 1]
```diff
@@ -20,7 +21,9 @@ public class Company {
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
 
+    @NotBlank
     private String companyName;
 
+    @NotNull
```
Comment (kurylek) [inline Java/E-montazysta/src/main/java/com/emontazysta/model/Company.java]: createdAt is generated on Company create- we don't pass it in body

File: Java/E-montazysta/src/main/java/com/emontazysta/model/Comment.java [hunk 2]
```diff
@@ -22,7 +21,9 @@ public class Comment {
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
 
+    @NotBlank
     private String content;
 
+    @NotBlank
     private Date createdAt;
 }
```
Comment (kurylek) [inline Java/E-montazysta/src/main/java/com/emontazysta/model/Comment.java:29]: Changes not related to Cu 862j4d59c

**Review by kurylek — CHANGES_REQUESTED** [review]
We should remove `@NotNull` from `private Date createdAt;` Company.java:28 - value is generated

**Review by kurylek — APPROVED** [review]

**Pull request merged**